plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/lookup/ToolItemLookup.java

```diff
@@ -87,7 +87,7 @@ public ToolItem getWorkbenchToolItem(Matcher<String> matcher) {
 		Shell swtShell = ShellLookup.getInstance().getWorkbenchShell();
 		String text = WidgetHandler.getInstance().getText(swtShell);
 		WidgetHandler.getInstance().setFocus(swtShell);
-		new WaitUntil(new ShellWithTextIsActive(text));
+		new WaitUntil(new ShellWithTextIsAvailable(text));
```

Comment:
On this place waiting for a ShellWithTextIsAvailable is quite useless. Shell is already available and in previous step its getting focused. Original wait condition waited for the shell to be really focused. I would replace those several lines with "new WorkbenchShell()"